exercises/run-length-encoding/run_length_encoding.go

```diff
@@ -0,0 +1,5 @@
+package run_length_encoding
```

Comment:
I thought that a stub for this one won't be needed for run-length-encoding, since raindrops is likely before this one and raindrops is the first one without a stub.  Is that correct @robphoenix and @tleen ?

Reply:
Not really sure yet where this will land in nextercism. But it won't be in core and probably after raindrops. It's not like we can't have stubs after that, but we may not want to. We can modify this exercise afterwards when we place it?

Reply:
I think it's best to remove the stub file. I don't think this exercise will be before `raindrops` and we can see when v2 launches if we  need to add any stubs to later exercises.